README.md

```diff
@@ -16,29 +16,50 @@
 ## About the Project
 This Project serves the backend APIs required for [Real Dev Squad](https://realdevsquad.com/) web projects. This project is built in [Express.js](https://expressjs.com/).
 
-## Running the Project
-We are moving to yarn, to ensure that we use yarn , we are moving ahead with [Volta](https://docs.volta.sh/guide/#why-volta)
+## Prerequisites
+- The application uses [node-config](https://github.com/lorenwest/node-config)([documentation](https://github.com/lorenwest/node-config/wiki/Configuration-Files)) for managing config.
+- Create a new file: `config/local.js`. Override the required config values from `config/development.js` and `config/default.js` into `config/local.js`.
```

Comment:
Can we simply this even further? Override something from 2 files doesn't seem very beginner-friendly I guess? 🤔

Reply:
I think #500 will take care of this.